pkg/resourceinterpreter/customized/webhook/customized.go

```diff
@@ -70,7 +68,7 @@ func NewCustomizedInterpreter(informer genericmanager.SingleClusterInformerManag
 	}
 
 	cm.SetAuthenticationInfoResolver(authInfoResolver)
-	cm.SetServiceResolver(apiserver.NewClusterIPServiceResolver(serviceLister))
```

Comment:
The issue was introduced when adding the service resolver, right? (#2999)

Reply:
Ignore me.
Even after #2999, the import order guarantees `apiserver.NewClusterIPServiceResolver(serviceLister)` be initialized before `k8s.io/component-base/metrics/prometheus/workqueue`.